lib/components/table.vue

```diff
@@ -148,7 +148,7 @@ function recToString(obj) {
 
     return toString(keys(obj).reduce((o, k) => {
         // Ignore fields that start with _
-        if (!/^_/.test(k)) {
+        if (!(/^_/).test(k)) {
```

Comment:
Literal RegExpr shouldn't need wrapping in brackets.

If ES Lint is complaining, then try this:
```js
if (!(/^_/.test(k)))
```

Reply:
it works good with this `if (!(/^_/).test(k)) {`